Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM quay.io/openshift/origin-must-gather:latest
+FROM quay.io/openshift/origin-must-gather:latest as builder
+
+FROM centos:7
```

Comment:
what do we need from centos7?

Reply:
That’s the base image for node gather. All requires packages for collecting information are installed from.